packages/x-charts-pro/src/BarChartPro/BarChartPro.zoom.test.tsx

```diff
@@ -59,7 +60,7 @@ describe.skipIf(isJSDOM)('<BarChartPro /> - Zoom', () => {
     );
     expect(getAxisTickValues('x')).to.deep.equal(['A', 'B', 'C', 'D']);
 
-    const svg = document.querySelector('svg')!;
+    const svg = document.querySelector(CHART_SELECTOR)!;
```

Comment:
Had to do this because `querySelector` was selecting the measurement SVG instead of the chart's